docs/Measuring_Resonances.md

```diff
@@ -51,6 +51,13 @@ Note that on a bed slinger printer one must design 2 mounts: one for the
 toolhead and one for the bed, and run the measurements twice. See the
 corresponding [section](#bed-slinger-printers) for more details.
 
+**Attention:** neither the accelerometer nor the screws that hold it in
+place (if any) should not touch any metal parts of the printer.
+Basically, the mount must be designed such as to ensure the electrical
+isolation of the accelerometer from the printer frame. Failing to ensure
+that can create a ground loop in the system, and damage the electronics
+in the worst case scenario.
```

Comment:
Heh - I'm sure there's an even worse, "worse case scenario".  How about just `Failing to ensure
that can create a ground loop in the system that may damage the electronics.`

Reply:
That is probably true. OK, I've changed the wording per your suggestion.